src/Microsoft.Framework.Runtime.Sources/Impl/Constants.cs

```diff
@@ -9,6 +9,7 @@ internal static class Constants
         public const string BootstrapperFullName = "Microsoft .NET Execution environment";
         public const string DefaultLocalRuntimeHomeDir = ".dnx";
         public const string RuntimeShortName = "dnx";
+        public const string RuntimeLongName = "Microsoft " + RuntimeShortName;
```

Comment:
Does it matter that we now use lowercase 'dnx' for folder and environment variable names while the actual names are uppercase? Are we going to support bredcrumbs on other platforms (where it does)?